OpenGL/Constructs/VAO.cs

```diff
@@ -1,11 +1,13 @@
 ﻿using System;
+using System.Collections.Generic;
 
 #if USE_NUMERICS
 using System.Numerics;
 #endif
 
 namespace OpenGL
 {
+    [Obsolete("Use VAO<T1, T2> instead.")]
```

Comment:
I'd use the same Obsolete message as below, since this message doesn't explain why to use VAO<T1, T2>.

Reply:
Yup, I just meant to use the same type of Obsolete message for VAO<T1> that was used for the other non-generic element array VAOs.  Your change is perfect.